main.js

```diff
@@ -1,10 +1,62 @@
+function convertToBool(str) {
+  if (['true', 'false'].includes(str)) {
+    return str === 'true';
+  }
+}
+
 function toggleToDo(element){
 
-  var faClassName = element.querySelector(".fa");
+  var faClassName = element.getElementsByClassName("fa")[0];
```

Comment:
Please use `const` instead of var everywhere.

Reply:
Addressed with this commit; f1a80f1fdaa38cd64bf49d5c6bc33ec3ca4c7c1f